playwright-e2e/fixtures/create-fixture.ts

```diff
@@ -115,8 +118,15 @@ export const test = base.extend<CreateFixtures>({
     await use(new CaseDetails(page));
   },
 
+  childDetails: async ({ page }, use) => {
+    await use(new ChildDetails(page));
+  },
+
+  basePage: async ({ page }, use) => {
```

Comment:
is there a reason why we need basePage here?

Reply:
just pulling the branch now and removing from fixture to see - can't quite remember why i added it..

Reply:
yep, not needed -- deleted now